packages/api-openfin/window.js

```diff
@@ -2,4 +2,48 @@ if (!window.ssf) {
   window.ssf = {};
 }
 
-window.ssf.window = (...args) => window.open(...args);
+window.ssf.window = function(url, name, features) {
+  let newWindow;
+  const featureObject = parseFeaturesString(features);
+  const handleError = (error) => console.error('Error creating window: ' + error);
```

Comment:
I think we'll need to consider some way of allowing the user to handle errors themselves - allowing them to pass their own callback to handle the error, for example.